clj-refactor.el

```diff
@@ -149,6 +149,12 @@ as can be."
   :group 'cljr
   :type 'boolean)
 
+(defcustom cljr-eagerly-cache-macro-occurrences-on-startup t
```

Comment:
Why make this a defcustom?  I don't see any reason to ever turn this off.  Do you?